r10k.gemspec

```diff
@@ -23,7 +23,8 @@ Gem::Specification.new do |s|
   s.license  = 'Apache-2.0'
 
   s.add_dependency 'colored2',   '3.1.2'
-  s.add_dependency 'cri', '2.15.10'
+  s.add_dependency 'cri', '2.15.10' if RUBY_VERSION < '2.5.0'
+  s.add_dependency 'cri', '>= 2.15.10', '< 3' if RUBY_VERSION >= '2.5.0'
```

Comment:
This doesn't work. A gemspec is evaluated during build time, which means it will now depend on when `gem build` runs what the dependencies end up being.